boards/doc.txt

```diff
@@ -15,4 +15,52 @@
  * configuration of a controller and some external devices such as sensors or
  * radios. All aspects concerning configuration of GPIO pins, MCU clock and
  * device drivers should go into this module.
+ *
+## Guide to board pinouts
+
+The following are the steps to take to locate specific pinouts on boards.
+This is useful when connecting external hardware to RIOT supported boards.
+
+1. Check the board documentation to see if the pinout is available
+2. If the pinout is not available or up to date check the following:
+    - `boards/<TARGET_BOARD>/include/periph_conf.h` for board specific
+peripherals such as uart  or spi pins
+    - `boards/<TARGET_BOARD>/include/board.h` for GPIO specific pins such
```

Comment:
For variable part I would stick with already used (environment) variables, i.e. we never user `TARGET_BOARD` its either simply `BOARD` or `RIOT_BOARD` - don't introduce new names that will only confuse

Reply:
Yup